packages/terra-dev-site/src/layouts/_ContentLayout.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import MainContainer from '@cerner/terra-application/lib/main-container';
+import classNamesBind from 'classnames/bind';
+import Suspense from '../terra-application-temporary/shared/Suspense';
```

Comment:
a couple of these imports have been updated to look a the temporary implementation instead of terra-application.